tests/test_browser_debug_launcher.py

```
@@ -0,0 +1,257 @@
+    spec = FirefoxDebugLaunchSpec(
+        debug_url="ws://127.0.0.1:6000",
+        browser_path=str(browser_bin),
+        profile_path=tmp_path / "profile",
+        start_url="https://gengo.com/t/jobs/status/available/realtime",
+        port=6000,
+    )
+
+    with patch("gengowatcher.browser_debug_launcher.subprocess.Popen") as mock_popen:
+        try:
+            launch_managed_firefox_debug(spec)
+            assert False, "Expected locked Firefox build to be rejected"
```

Ruff (0.15.10)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)